### PR TITLE
Allow different radii at segments connections and update utils scripts

### LINKED ISSUE #4: Remove assumption that radius at end should meet next radius at start

### PR DESCRIPTION
**Pull request by JuxhinoKavaja**
Closes #4 and updates utils scripts to take curvature into account.

### CODE REVIEW

File: tracks/tracks.csv [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-00_var_speed_limit_110,110,140,0.0,0.0,48531.0,10000.0,25000.0,3,2
-00_var_speed_limit_120,120,140,0.0,0.0,48531.0,10000.0,25000.0,3,2
-00_var_speed_limit_wind,50,120,0.0,0.0,20000.0,1000.0,7000.0,6,2
-CH_Fribourg_Bern,40,140,-16.9,14.1,31240.7,15.8,922.6,132,2
-CH_Stadelhofen_Altstetten,80,125,-38.0,28.0,5790.0,10.0,430.0,223,4
-CN_Songjiazhuang_Yizhuang,50,84,-24.0,24.0,22728.0,1.0,700.0,89,14
-SE_Vasteras_Kolback,110,200,-16.7,10.8,19305.4,106.8,1279.6,51,2
+ID,Min speed limit [km/h],Max speed limit [km/h],Min gradient [permil],Max gradient [permil],Min (abs) radius [m],Max (abs) radius [m],Length [m],Min interval [m],Max interval [m],Num intervals [-],Num stops [-]
+00_reference,140,140,0.0,0.0,inf,inf,48531.0,48531.0,48531.0,1,4
+00_stationX_stationY,80,125,0.0,0.0,340.1,inf,29556.1,15.9,1739.9,244,2
```
Comment (dkouzoup) [inline tracks/tracks.csv]: @juxhino92 do the changes here make sense? I would expect only the new values added + changes in min interval, max interval, num intervals (if applicable)

File: utils/printTracks.py [hunk 2]
```diff
@@ -24,14 +24,26 @@ def printTracks(tracksDir, filename=None):
             numStops = len(data['stops']['values'])
 
             speedLimitValues = [v[1]*(3.6 if data['speed limits']['units']['velocity'] == 'm/s' else 1) for v in data['speed limits']['values']]  # km/h
-            gradientValues = [v[1] for v in data['gradients']['values']]  # permil
             speedLimitPositions = [v[0]*(1e3 if data['speed limits']['units']['position'] == 'km' else 1) for v in data['speed limits']['values']]  # m
-            gradientPositions = [v[0]*(1e3 if data['gradients']['units']['position'] == 'km' else 1) for v in data['gradients']['values']]  # m
 
-            positions = sorted(set(speedLimitPositions + gradientPositions + [length]))
+            if 'gradient' in data:
+                gradientValues = [v[1] for v in data['gradients']['values']]  # permil
```
Comment (dkouzoup) [inline utils/printTracks.py:31]: I am being pedantic here but please introduce empty lines in the nested if statements (and try/except) as in the rest of the code

**Comment by dkouzoup** [comment]
also partially addresses #3

File: tracks/tracks.csv (hunk 1 above)
Comment (JuxhinoKavaja) [reply tracks/tracks.csv]: Thanks for spotting the inconsistency!

File: utils/printTracks.py [hunk 3]
```diff
@@ -58,4 +76,4 @@ def printTracks(tracksDir, filename=None):
 
     tracksDir = "../tracks"
 
-    printTracks(tracksDir, filename=None)
+    printTracks(tracksDir, filename='tracks.csv')
```
Comment (dkouzoup) [inline utils/printTracks.py]: please leave to None since we don't want to actively change the track.csv file every time

**Review by dkouzoup — APPROVED** [review]
Thanks!

**Pull request merged**